packages/components/src/tab-panel/style.scss

```diff
@@ -0,0 +1,43 @@
+.components-tab-panel__tabs {
```

Comment:
I was surprised that the "TabPanel" component doesn't include any styling by default. I added basic styling here, I think this is needed regardless of this PR but may require a dev note.

Reply:
There is an issue filed: https://github.com/WordPress/gutenberg/issues/13587